var/spack/repos/builtin/packages/r-pbdzmq/package.py

```diff
@@ -17,10 +17,14 @@ class RPbdzmq(RPackage):
     functions compatible with 'rzmq' are also provided."""
 
     homepage = "http://r-pbd.org/"
-    url      = "https://cran.r-project.org/src/contrib/pbdZMQ_0.2-4.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/pbdZMQ"
+    url      = "https://cloud.r-project.org/src/contrib/pbdZMQ_0.2-4.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/pbdZMQ"
 
+    version('0.3-3', sha256='ae26c13400e2acfb6463ff9b67156847a22ec79f3b53baf65119efaba1636eca')
+    version('0.3-2', sha256='ece2a2881c662f77126e4801ba4e01c991331842b0d636ce5a2b591b9de3fc37')
     version('0.2-4', 'e5afb70199aa54d737ee7a0e26bde060')
 
+    depends_on('r@3.0.0:', when='@:0.2-5', type=('build', 'run'))
+    depends_on('r@3.2.0:', when='@0.2-6:', type=('build', 'run'))
     depends_on('r-r6', type=('build', 'run'))
```

Comment:
I don't see this dependency listed anywhere

Reply:
It was needed up until version 0.2-6. I added that constraint.